test-core-development.cfg

```diff
@@ -0,0 +1,13 @@
+# Tests whether the lastest opengever.core development still works with a policy.
+# Should be extended within the policy package.
+
+[buildout]
+extends =
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-plone-4.3.2.cfg
```

Comment:
If we update the Plone version in http://kgs.4teamwork.ch/release/opengever/develop, we'll also need to remember to bump it here. Maybe a big fat comment in the `develop` KGS would be in order?